zhaquirks/sonoff/snzb03p.py

```diff
@@ -0,0 +1,64 @@
+"""Sonoff SNZB-03P - Zigbee motion sensor."""
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import EntityPlatform, EntityType, NumberDeviceClass, QuirkBuilder
+from zigpy.quirks.v2.homeassistant import UnitOfTime
+import zigpy.types as t
+from zigpy.zcl.clusters.measurement import OccupancySensing
+from zigpy.zcl.clusters.security import IasZone
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class LastIlluminationState(t.enum8):
+    """Last measureed state of illumination enum."""
+
+    Dark = 0x00
+    Light = 0x01
+
+
+class SonoffIlluminationCluster(CustomCluster):
+    """Custom Sonoff illumination cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        last_illumination_state = ZCLAttributeDef(
+            id=0x2001,
+            type=LastIlluminationState,
+            access="rp",
+            is_manufacturer_specific=True,
+        )
+
+    @property
+    def _is_manuf_specific(self):
+        return False
+
+
+(
+    QuirkBuilder("eWeLink", "SNZB-03P")
+    .replaces(SonoffIlluminationCluster)
+    .removes(IasZone.cluster_id)  #  IasZone cluster 0x0500: remove motion detection
```

Comment:
So the `IasZone` cluster doesn't do anything at all?
Can we use `.prevent_default_entity_creation(endpoint_id=1, cluster_id=IasZone.cluster_id)` here instead?
It's a new functionality that doesn't entirely remove the cluster for us, just disables the auto discovery for ZHA entities on that cluster.

Reply:
Yes, cluster is always "unavalaible".